### ISSUE TITLE
BroadcastFilters per endpoint

### ISSUE THREAD

**Issue by tbolis** [Atmosphere/atmosphere#1889]
Please see discussion from groups: [discussion](https://groups.google.com/forum/#!topic/atmosphere-framework/xsCy0oWNlrQ)

Instead of globally setting broadcastFilters for all Atmosphere endpoints, broadcastFilters per endpoint should be supported

Thank you

**Comment by jfarcand** [comment]
@tbolis In case you want to help, the idea is to set the collection on the `AtmosphereHandlerWrapper` inner classe of `AtmosphereFramework` and the re-use that.

**Comment by tbolis** [comment]
Hi, made some tries but with no luck (and due to extra limited time). If you can give me some more details... is there any similar way to follow? (e.g. interceptors)

**Comment by jfarcand** [comment]
@tbolis Yes, take a look at `AtmosphereHandlerWrapper` and add a filed for the `broadcasterFilers`

**Comment by jfarcand** [comment]
Closing as won't fix — this issue references APIs, containers, or patterns from the pre-4.x era that no longer exist in Atmosphere 4.x. The 4.x rewrite addresses these concerns architecturally.
